### ISSUE TITLE
Bug on DS: System inserting old ROW Usage Fees

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#10018]
### AMANDA is inserting FY21 ROW Usage Fees on the DS folder. 
- Example RSN 12743283

- ROW Usage Fee only; not multiple fees (50150)
- Correct fee: 6.05 per structure per day
- Incorrect (current) 5.50 per structure per day

**Comment by TracyLinder** [comment]
**Update** This is occuring on all DS folders. 

- ROW Usage Fee only; not multiple fees
- Correct fee: 6.05 per structure per day
- Incorrect (current) 5.50 per structure per day

**Comment by TracyLinder** [comment]
@Nadin-Nader this is ready to be assigned.

**Comment by SubhaShroff** [comment]
@TracyLinder  @Nadin-Nader There are three ROW usage fees configured, as follows. Looks like each fee entry is used for different DS subtypes, not sure though. I was about to change the value in the first row to 6.05. But, want to make sure that rate(5.5) is not used by some other DS/EX subtype permit.

![image](https://user-images.githubusercontent.com/51680715/191575622-f9bfbac0-94b5-4150-9393-28cb4c5649e9.png)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  [SCTASK0250412](https://atx.servicenowservices.com/sc_task.do?sys_id=11d3365e1b86d91091528407624bcb34&sysparm_record_target=sc_task&sysparm_record_row=1&sysparm_record_rows=1&sysparm_record_list=request_item%3D05d3f25e1b86d91091528407624bcbce%5EORDERBYnumber) is submitted for DEV/TEST merge after SB merge is done.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  @TracyLinder  This is merged into DEV/TEST. Ready for UAT.

**Comment by TracyLinder** [comment]
### Approved for PROD
- Test case 2021-161058
FYI @SubhaShroff

**Comment by Nadin-Nader** [comment]
According to Kensley I. The Driveway/Sidewalk Application fee is not calculating correctly. It is showing $196 and its supposed to be $260
![MicrosoftTeams-image (18).png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/5e3cb5b3-8215-47e1-8309-554159ae44be)

**Comment by SubhaShroff** [comment]
@Nadin-Nader  I investigated it through, this one was all good through TEST. In PROD, the amount was changed back to old amount due to merge conflict. Informed George. He will be fixing it.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  This is fixed for flat fee. Please ask the user to test the structured fee as well. Thanks.

**Comment by SubhaShroff** [comment]
@Nadin-Nader , I submitted SCTASK0250412 for DEV/TEST/PROD for DS inspection fee error(above mentioned).

**Comment by SubhaShroff** [comment]
@Nadin-Nader  deployed the above request in DEV/TEST. This evening it will be seen on PROD.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  The above deployment is done. DS Inspection Fee insertion must be working fine now on PROD.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0254099 is submitted for DEV/TEST/PROD deployment of Sidewalk Inspection fee error fix.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  SCTASK0254099-This is deployed on DEV/TEST/PROD. Sidewalk Inspection fee error should be resolved everywhere now.

**Comment by Nadin-Nader** [comment]
This is still Happening

**Comment by TracyLinder** [comment]
**UAT in PROD:** ROWID 13033817 created & ROW usage fees added. Waiting on ROW to confirm if the fee is correct or not.

**Comment by SubhaShroff** [comment]
> This is still Happening

@Nadin-Nader  for Sidewalk Inspection Fee error, I submitted SCTASK0255330 DEV/TEST deployment request.

**Comment by SubhaShroff** [comment]
@Nadin-Nader  The above deployment is done and I tested. Sidewalk Inspection fee error fix I tested on TEST and no error displays but if the users can test for the amounts inserted and the fee name & surcharge is correct.

**Comment by Nadin-Nader** [comment]
This has been resolved. The Sidewalk Inspection fee is no longer getting system errors. Closing
